File: output/deepseek_chat/codenet/algo_based_translation/python/java/s636719275.java

import java.util.Scanner;

public class s636719275 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.nextLine();
        String t = scanner.nextLine();
        
        int ans = 1000000;
        for (int i = 0; i < s.length(); i++) {
            String a = s.substring(i);
            if (a.length() >= t.length()) {
                String candidate = a.substring(0, t.length());
                int currentCheck = check(t, candidate);
                ans = Math.min(ans, currentCheck);
            }
        }
        System.out.println(ans);
    }
    
    public static int check(String a, String b) {
        int s = 0;
        for (int i = 0; i < a.length(); i++) {
            if (a.charAt(i) != b.charAt(i)) {
                s++;
            }
        }
        return s;
    }
}

File: dataset/codenet/Python/Code/s636719275.py
s = input()
t = input()

def check(a,b):
    s = 0
    for i,j in zip(a,b):
        s += i!=j
    return s
ans = 10**6
for i in range(len(s)):
    a = s[i:]
    if len(a) >= len(t):
        ans = min(ans,check(t,a[:len(t)]))
print(ans)